exporter.go

```diff
@@ -199,6 +222,11 @@ For NGINX, the stub_status page must be available through the URI. For NGINX Plu
 func main() {
 	flag.Parse()
 
+	constLabelMap, err := parseConstLabels(*constLabels)
+	if err != nil {
+		log.Fatalf("Parsing const labels failed: %v", err)
+	}
+
```

Comment:
what about labels for `buildInfoMetric` metric below?

Reply:
I thought it was not necessary, adding now.